src/SaveHandler/Cache.php

```diff
@@ -10,19 +10,25 @@
  * Cache session save handler
  *
  * @see ReturnTypeWillChange
+ *
+ * @final
  */
 class Cache implements SaveHandlerInterface
 {
     /**
      * Session Save Path
      *
+     * @deprecated This property will be removed in version 3.0.
```

Comment:
Adding a reason would be good, like:

```suggestion
     * @deprecated This property will no longer be needed in future and will therefore be removed in version 3.0.
```